plugins/leemons-plugin-assignables/frontend/src/components/ActivityHeader/index.js

```diff
@@ -95,6 +104,13 @@ export default function ActivityHeader({
 
   const { classes } = useTotalLayoutStyles();
 
+  const goToAssignmentDetail = () => {
+    const url =
+      instance?.assignable?.roleDetails?.dashboardUrl ||
+      '/private/assignables/details/:id'.replace(':id', instance.id);
```

Comment:
The dashboardURL also has a :id in its value

```suggestion
    const url =
      (instance?.assignable?.roleDetails?.dashboardUrl ||
      '/private/assignables/details/:id').replace(':id', instance.id);
```

Reply:
Ok, proceed to fix this. Thanks!